src/app/sidebar/components/CardDropDown.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { Box, Collapse, Flex, Heading, IconButton } from "@chakra-ui/react";
+import styled from "@emotion/styled";
+import { ChevronDownIcon } from "@chakra-ui/icons";
+import { Colors } from "../shared/styles";
+
+export interface CardProps {
+    /*
+    * The subject
+    * Ex/ CSC
+    */
+    subject: string;
+    /*
+    * The subject title
+    * Ex/ Computer Science
+    */
+    title: string;
+    /*
+    * The list of courses for this subject
+    */
+    courseCards: JSX.Element[];
+}
+
+export function CardDropDown(props: CardProps): JSX.Element {
+    const [isDisplayed, setDisplayed] = useState(false);
```

Comment:
I'm not sure if we want this component to hold the state. I think maybe that should be reflected in the main siderbar component. I think exposing a `onClick/onChange` etc. optional prop instead which would pass in what the user clicked (`pid`) would be better.

Reply:
We talked about this offline - keeping it as state